packages/theme/src/cli/services/dev.ts

```diff
@@ -165,6 +175,48 @@ export async function dev(options: DevOptions) {
   ])
 }
 
+export function createKeypressHandler(
```

Comment:
Maybe I'm not reading something right but I don't see this function actually used anywhere?